src/scaffoldfitter/fitter.py

```diff
@@ -1196,12 +1195,12 @@ def calculateGroupDataProjections(self, fieldcache, group, dataGroup, meshGroup,
         dataProportion = activeFitterStepConfig.getGroupDataProportion(groupName)[0]
         centralProjection = activeFitterStepConfig.getGroupCentralProjection(groupName)[0]
         if centralProjection:
-            # get geometric centre of dataGroup
-            dataCentreField = self._fieldmodule.createFieldNodesetMean(dataCoordinates, dataGroup)
-            result, dataCentre = dataCentreField.evaluateReal(fieldcache, dataCoordinates.getNumberOfComponents())
-            if result != RESULT_OK:
-                print("Error: Centre Groups projection failed to get mean coordinates of data for group " + groupName)
+            # use centre of bounding box as middle of data; previous use of mean was affected by uneven density
+            minDataCoordinates, maxDataCoordinates = evaluate_field_nodeset_range(dataCoordinates, dataGroup)
+            if minDataCoordinates is None:
```

Comment:
Is there any possibility for maxDataCoordinates to be None?

Reply:
Either both are None, or both are valid range extents, hence just checking one.

Reply:
I'm testing both now, see commit below.